app/src/main/java/com/example/myapplication/ui/theme/Typography.kt

```diff
@@ -0,0 +1,85 @@
+package com.example.myapplication.ui.theme
+
+import androidx.compose.material3.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import com.example.myapplication.R
+
+val Pretendard = FontFamily(
+    Font(R.font.pretendard_regular, FontWeight.Normal),
+    Font(R.font.pretendard_medium, FontWeight.Medium),
+    Font(R.font.pretendard_semibold, FontWeight.SemiBold),
+    Font(R.font.pretendard_bold, FontWeight.Bold)
+)
+
+val body_med_16 = TextStyle(
+    fontFamily = Pretendard,
+    fontWeight = FontWeight.Medium,
+    fontSize = 16.sp
+)
+
+val body_semi_14 = TextStyle(
+    fontFamily = Pretendard,
+    fontWeight = FontWeight.SemiBold,
+    fontSize = 14.sp
+)
+
+val body_reg_14 = TextStyle(
+    fontFamily = Pretendard,
+    fontWeight = FontWeight.Normal,
+    fontSize = 14.sp
+)
+
+val cap_bold_12 = TextStyle(
+    fontFamily = Pretendard,
+    fontWeight = FontWeight.Bold,
+    fontSize = 12.sp
+)
+
+val cap_semi_12 = TextStyle(
+    fontFamily = Pretendard,
+    fontWeight = FontWeight.SemiBold,
+    fontSize = 12.sp
+)
+
+val cap_med_12 = TextStyle(
+    fontFamily = Pretendard,
+    fontWeight = FontWeight.Medium,
+    fontSize = 12.sp
+)
+
+val cap_semi_10 = TextStyle(
+    fontFamily = Pretendard,
+    fontWeight = FontWeight.SemiBold,
+    fontSize = 10.sp
+)
+
+val cap_medi_10 = TextStyle(
```

Comment:
여기 다른 거랑 맞추려면 medi -> med로 수정해야 할 거 같아용
```suggestion
val cap_med_10 = TextStyle(
```

Reply:
꼼꼼한 지적 감사합니다 바로 고쳤어요 👍🏻🥹